package/gnss_convertors/gnss_convertors.mk

```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-GNSS_CONVERTORS_VERSION = v0.3.64
+GNSS_CONVERTORS_VERSION = 6d5f0896caad82d16dcb401196997c39b8434f76
```

Comment:
this commit is https://github.com/swift-nav/gnss-converters/pull/135, can we merge and point to a release?